grafana/dashboards/cmode/s3ObjectStorage.json

```diff
@@ -311,6 +311,38 @@
               {
                 "id": "displayName",
                 "value": "Volume"
+              },
+              {
+                "id": "links",
+                "value": [
+                  {
+                    "targetBlank": true,
+                    "title": "",
+                    "url": "/d/cdot-volume/ontap-volume?orgId=1&${Datacenter:queryparam}&${Cluster:queryparam}&${SVM:queryparam}&${__url_time_range}&var-Volume=${__value.raw}"
+                  }
```

Comment:
This link may not work as this volumes starts with fg_xxx and volume dashboard don't show them.